scripts/coverage.ts

```diff
@@ -0,0 +1,65 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as istanbul from 'istanbul-lib-coverage';
+import * as libReport from 'istanbul-lib-report';
+import * as reports from 'istanbul-reports';
+
```

Comment:
nit:
can you add a line or two describing this file
"Generates code coverage for each package to packages/<pkg>/coverage/coverage-final" or what you feel is appropriate